code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+	short_name = "Bishop"
+	acronym = "BC"
+	desc = "Bishop's focus is on high-class, stylish cybernetics. A favorite among transhumanists (and a b'te noire for \
+	bioconservatives), Bishop manufactures not only prostheses but also brain augmentation, synthetic organ replacements, \
+	and odds and ends like implanted wrist-watches. Their business model tends towards smaller, boutique operations, giving \
+	it a reputation for high price and luxury, with Bishop cyberware often rivalling Vey-Med's for cost. Bishop's reputation \
+	for catering towards the interests of human augmentation enthusiasts instead of positronics have earned it ire from the \
+	Positronic Rights Group and puts it in ideological (but not economic) competition with Morpheus Cyberkinetics."
+	history = ""
+	work = "cybernetics and augmentation manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("BTV" = "transportation")
+	destination_names = list()
+
+/datum/lore/organization/tsc/morpheus
+	name = "Morpheus Cyberkinetics"
+	short_name = "Morpheus"
+	acronym = "MC"
+	desc = "The only large corporation run by positronic intelligences, Morpheus caters almost exclusively to their sensibilities \
+	and needs. A product of the synthetic colony of Shelf, Morpheus eschews traditional advertising to keep their prices low and \
+	relied on word of mouth among positronics to reach their current economic dominance. Morpheus in exchange lobbies heavily for \
+	positronic rights, sponsors positronics through their Jans-Fhriede test, and tends to other positronic concerns to earn them \
+	the good-will of the positronics, and the ire of those who wish to exploit them."
+	history = ""
+	work = "cybernetics manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("MTV" = "freight")
+	// Culture names, because Anewbe told me so.
+	ship_names = list(
+		"Nervous Energy",
+		"Prosthetic Conscience",
+		"Revisionist",
+		"Trade Surplus",
+		"Flexible Demeanour",
+		"Just Read The Instructions",
+		"Limiting Factor",
+		"Cargo Cult",
+		"Gunboat Diplomat",
+		"A Ship With A View",
+		"Cantankerous",
+		"Never Talk To Strangers",
+		"Sacrificial Victim",
+		"Unwitting Accomplice",
+		"Bad For Business",
+		"Just Testing",
+		"Size Isn't Everything",
+		"Yawning Angel",
+		"Liveware Problem",
+		"Very Little Gravitas Indeed",
+		"Zero Gravitas",
+		"Gravitas Free Zone",
+		"Absolutely No You-Know-What",
+		"Existence Is Pain",
+		"Screw Loose",
+		"Of Course I Still Love You",
+		"Limiting Factor",
+		"So Much For Subtley",
+		"Unfortunate Conflict Of Evidence",
+		"Prime Mover",
+		"Boo!",
+		"Reasonable Excuse",
+		"Honest Mistake",
+		"Appeal To Reason",
+		"My First Ship II",
+		"Hidden Income",
+		"Anything Legal Considered",
+		"New Toy",
+		"Me, I'm Always Counting",
+		"Just Five More Minutes",
+		"Are You Feeling It",
+		"Great White Snark",
+		"No Shirt No Shoes",
+		"Callsign"
+	)
+	destination_names = list(
+		"a dockyard on New Canaan"
+	)
+
```

Comment:
/datum/lore/organization/tsc/shellguard
	name = "Shellguard Munitions"
	short_name = "Shellguard"
	acronym = "SM"
	desc = "The brainchild of a colonial war veteran, Shellguard Munitions is an arms manufacturer and private military contractor specializing in anti-synthetic weapon systems and platforms. Initially a smaller private military force only serving frontier colonies, Shellguard Munitions has become a household name due to its involvement in resolving the Haverick AI crisis in 2552. Using its recently earned fame, the company has made a successful foray into the market of robotics and is highly regarded for the toughness and reliability of their hardware. Despite being frequently contracted by the Trans-Solar Federation, Shellguard Munitions is known to sell their services to the highest corporate bidder."
	history = ""
	work = "anti-synthetic arms manufacturer and PMC"
	headquarters = "Colony of Haverick"
	motto = ""

	ship_prefixes = list("BTS-T" = "transportation")
	destination_names = list()